Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV PROXY_HOST=localhost \
     DISPLAY=:99 \
     GEOMETRY=1360x1020x16
 
-RUN pip install git+https://github.com/webrecorder/pywb@patch-work
+RUN pip install git+https://github.com/TheDataShed/pywb@add_s3_uploader
```

Comment:
Was the `--use-feature=2020-resolver` flag necessary in the end?

Reply:
I was going to try and fix the warnings and dependency issues in another branch